src/options/decorator.options.ts

```diff
@@ -1,3 +1,6 @@
+// eslint-disable-next-line @typescript-eslint/ban-types
+export type ValidateOptions = Function;
```

Comment:
The type definition in `class-validator` is:

```ts
import { ValidationOptions } from '../ValidationOptions';
/**
 * Registers custom validator class.
 */
export declare function ValidatorConstraint(options?: {
    name?: string;
    async?: boolean;
}): (target: Function) => void;
/**
 * Performs validation based on the given custom validation class.
 * Validation class must be decorated with ValidatorConstraint decorator.
 */
export declare function Validate(constraintClass: Function, validationOptions?: ValidationOptions): PropertyDecorator;
export declare function Validate(constraintClass: Function, constraints?: any[], validationOptions?: ValidationOptions): PropertyDecorator;
```

I'm omitting the secondary `validatorOptions` and `constraints` inputs here for now, but we can generalize `ValidateOptions` to be a union type if we find a need for these extra options.